controllers/subscription/subscription_controller.go

```diff
@@ -186,16 +188,26 @@ func (r *SubscriptionReconciler) HandleUpgrades(ctx context.Context, rhmiSubscri
 
 		return ctrl.Result{}, nil
 	}
+	log.Infof("RHMI subscription", l.Fields{"StartingCSV": rhmiSubscription.Spec.StartingCSV, "InstallPlanRef": rhmiSubscription.Status.InstallPlanRef})
```

Comment:
can we change the log message here to something more meanful? Checking the RMI subscription or verifying the fields in the subscription?